### ISSUE TITLE
Difficulty updating mamba from 0.25.0 (with conda 22.9.0) to 0.27.0

### ISSUE THREAD

**Issue by corneliusroemer** [mamba-org/mamba#2003]
I had difficulties updating my environment from using mamba 0.25.0 to 0.27.0

This may have something to do with the temporary pinnings that were introduced because mamba 0.25.0 is incompatible with conda 22.9.0

Here is how I resolved it:
0. Activate base environment: `conda activate base`
1. Downgrade conda to v4: `mamba install conda=4`
2. Update mamba: `mamba update mamba`
3. Update conda: `mamba update conda`

Not sure whether there's an easier way to do this, but this could help others who have the same issue I had, same as: https://github.com/mamba-org/mamba/issues/1950 https://github.com/mamba-org/mamba/issues/1969

**Comment by wolfv** [comment]
Hmm, it seems strange, because `conda=22.09...` can happily coexist with `mamba=0.27`. Sorry for the issues!

**Comment by corneliusroemer** [comment]
No worries @wolfv - I'm not quite sure how this happened, whether it's reproducible or not. I resolved using the steps above which can possibly help others in the same situation. But I failed to reproduce it just now after downgrading to mamba=0.25. Maybe it was because I was on build number 0. Whatever.

Thanks a lot for the great work on bringing conda into the 21st century, I just noticed the docs, amazing stuff, much more readable than conda docs. Can't wait to read them cover to cover, to finally _get_ conda/mamba: https://mamba.readthedocs.io/en/latest/advanced_usage/more_concepts.html
